### ISSUE TITLE
does anyone know what that theme is?

### ISSUE THREAD

**Issue by lucasfellipe3g** [ThePrimeagen/init.lua#128]
hello! does anyone know what that theme is?

![CleanShot 2024-08-26 at 13 23 39@2x](https://github.com/user-attachments/assets/e3a8d063-a8f3-41a8-89e5-5d82ae93d9e7)

**Comment by noreox** [comment]
The theme is called "Tokyonight".
To change the theme, you need to edit the file along the path: yourPathTo/nvim/lua/theprimeagen/lazy/colors.lua

For some reason, the Tokyonight theme appears twice in this file, but the default Tokyonight theme does not work, so you will need to move the "ColorMyPencils()" function from one place to another, in this case from the first original Tokyonight theme settings, to the second Tokyonight theme settings, moving the function mentioned earlier, after the line "config = function()"

**Comment by lucasfellipe3g** [comment]
> The theme is called "Tokyonight". To change the theme, you need to edit the file along the path: yourPathTo/nvim/lua/theprimeagen/lazy/colors.lua
> 
> For some reason, the Tokyonight theme appears twice in this file, but the default Tokyonight theme does not work, so you will need to move the "ColorMyPencils()" function from one place to another, in this case from the first original Tokyonight theme settings, to the second Tokyonight theme settings, moving the function mentioned earlier, after the line "config = function()"

are u sure? doesn't look like this theme

![CleanShot 2024-09-04 at 16 24 29@2x](https://github.com/user-attachments/assets/ed75df8e-27d3-4c33-8c04-aa903e8a27e8)

**Comment by noreox** [comment]
Yeah, I guess that's it, but it looks like WezTerm uses a different color scheme (or something) that doesn't seem to be directly related to neovim. 
Primeagen itself uses Alacritty as a terminal emulator, where the color scheme is different from many by default. 

But in my case, this solution helped me get the result from the original screenshot
![2024-09-04_23-05-20_screenshot](https://github.com/user-attachments/assets/0f21f6d8-4b2e-49e5-8fc5-ca97cb5bc234)

**Comment by noreox** [comment]
I finally found the original name of the theme, it is called "rose-pine” (https://github.com/rose-pine/neovim)

It's just under the "Tokyonight" theme setting, you just need to do the same as before (move the function)

**Comment by mrayushmehrotra** [comment]
it is rose-pine theme
